subdomains.json

```diff
@@ -38,6 +38,7 @@
   "piyush": "officialpiyush.github.io",
   "richie": "richiesuper.github.io",
   "rogerp": "rogerpanza.github.io",
+  "saz": "saz-io.github.io"
```

Comment:
You are missing a comma at the end of the key-value

Reply:
updated.